packages/dashboard/src/features/analytics/pages/SessionReplayPage.tsx

```
@@ -1,11 +1,11 @@
 import { useMemo, useState } from 'react';
 import {
   EmptyStateIllustration,
-  ErrorState,
   LoadingState,
   PaginationControls,
   TableHeader,
 } from '#components';
```

ESLint

[error] Replace `⏎··EmptyStateIllustration,⏎··LoadingState,⏎··PaginationControls,⏎··TableHeader,⏎` with `·EmptyStateIllustration,·LoadingState,·PaginationControls,·TableHeader·`

(prettier/prettier)